src/compiler/emitter.ts

```diff
@@ -841,10 +843,15 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
                     }
                     else {
                         if (i || leadingComma) {
-                            write(", ");
+                            write(",");
+							
+							let comments = getLeadingCommentsToEmit(node);
+							
+							if (!comments || !comments[0].hasTrailingNewLine) {

```

Comment:
the variable `comments` contains data like this `[ { pos: 131, end: 177, hasTrailingNewLine: true, kind: 3 } ]`. That's why I used `hasTrailingNewLine`.